usort/__init__.py

```diff
@@ -3,7 +3,19 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+# noqa: F401
+
 try:
-    from .version import version as __version__  # noqa: F401
+    from .version import version as __version__
 except ImportError:
     __version__ = "dev"
+
+from .api import usort_bytes, usort_path, usort_stdin, usort_string
+
+# DEPRECATED: preserve old api, will be removed by 1.0
```

Comment:
It should be safe to have `usort.sorting` do the imports from `.api`, why do you need to do assignment here? This is a little weird.

Reply:
That causes a circular import because usort_string needs something from sorting.

Reply:
Then I'd prefer usort/sorting.py contain wrappers so the temporary existence, and future deletion, is more obvious:

```py
# DEPRECATED: preserve old api, will be removed by 1.0
def usort_bytes(...):
  from .api import usort_bytes
  return usort_bytes(...)
```